res/schema/migrations/0002-add-metadata-dialog-tables.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS metadata (
+  hash VARCHAR(64) PRIMARY KEY NOT NULL,
+  metadata TEXT
+);
+
+CREATE TABLE IF NOT EXISTS entity (
+  id                INTEGER PRIMARY KEY NOT NULL,
+  context           VARCHAR(36)         NOT NULL REFERENCES execution (context),
+  kind              VARCHAR(256)        NOT NULL DEFAULT '',
+  name              VARCHAR(1024)       NOT NULL DEFAULT '',
+  metadata_hash     VARCHAR(64)         NOT NULL REFERENCES metadata (hash),
+  UNIQUE (context, kind, name)
+);
+
+CREATE TABLE dialog (
```

Comment:
IF NOT EXISTS as well, to ensure avoiding problems, when it already exists

Reply:
good catch. updated